kasa/cli.py

```diff
@@ -427,7 +423,7 @@ async def print_unsupported(unsupported_exception: UnsupportedDeviceException):
 
     echo(f"Discovering devices on {target} for {discovery_timeout} seconds")
 
-    async def print_discovered(dev: SmartDevice):
+    async def print_discovered(dev: iot.Device):
```

Comment:
So this PR is just the re-org and we haven't defined the `kasa.Device` interface yet so this needs to `iot.Device` until we add `update` etc to the base class. This will be the same answer for a few of the other comments below.